index.html

```diff
@@ -14,22 +14,30 @@
 </head>
 
 <body>
-    <header class="flex">
-        <div class="nav-bar flex">
+    <header class="">
+        <div class="nav-bar">
             <div>
                 <div class="topnav">
                     <a class="logo" href="#">My Logo</a>
                     <div class="nav-right ">
-                        <a id="menu-button" class="item-navigation" href="/# "><i class="fa fa-bars"></i></a>
-                        <a class="item-navigation hidden" href="#home ">Portfolio</a>
+                        <a id="menu-button" class="item-navigation" href="#"><i class="fa fa-bars"></i></a>
+                        <a class="item-navigation hidden" href="#card">Portfolio</a>
                         <a class="item-navigation hidden" href="#about ">About</a>
                         <a class="item-navigation hidden" href="#contact ">Contact</a>
                     </div>
                 </div>
             </div>
+            <div id="mobile-menu">
+                <a id="close" href="#"><i class="fa fa-close"></i></a>
```

Comment:
- [ ] Please make sure that when users click on menu item, the mobile menu disappears. In your case it stays on the top of the page.